uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/modules/shared-settings.spec.js

```diff
@@ -35,7 +35,7 @@ define([
       });
 
       it('contains the list of allowed mime types', function () {
-        expect(sharedSettings.fileUploader.allowedMimeTypes).toBeDefined();
+        expect(sharedSettings.fileUploader.mimeTypesMap).toBeDefined();
```

Comment:
Please compare `sharedSettings.fileUploader.mimeTypesMap` with some object instead of calling `toBeDefined`.